### PR TITLE
Remove specs without open changes for git-sync

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun @ricmatsui 

Implements parts of #506

### CODE REVIEW

File: features/git-rename-branch/non_feature_branch/with_remote_origin/with_tracking_branch/in_sync.feature [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Feature: git rename-branch: renaming a non-feature branch with a tracking branch
```
Comment (charlierudolph) [inline features/git-rename-branch/non_feature_branch/with_remote_origin/with_tracking_branch/in_sync.feature]: This doesn't appear to belong to this PR

**Comment by charlierudolph** [comment]
:ship: after comment is addressed

**Comment by kevgo** [comment]
Yes, that's a leftover from the rename-branch PR that must somehow slipt in here. Since that PR also got the green light, I'm going to submit it as part of this one.

**Comment by kevgo** [comment]
I'm sorry for the files from the wrong branches. I think its because I have been using Dropbox to sync my local Git Town setup between computers (a pre-Git Town setup that I used to write Git Town initially). Apparently with such large changes the synchronization takes a while, and some files show up delayed in the workspace. 

I am no longer using Dropbox for code any longer, so this shouldn't be an issue any longer. Good that you caught this issue @charlierudolph! Thanks to Git Town getting really awesome, I will now sync-all the full workspace with GitHub before switching machines!

**Pull request closed without merging**